cmd/crates/soroban-spec-tools/src/lib.rs

```diff
@@ -1185,7 +1193,7 @@ impl Spec {
             ScType::I32 => Some("-1".to_string()),
             ScType::Bool => Some("true".to_string()),
             ScType::Symbol => Some("\"hello\"".to_string()),
-            ScType::Status => Some("Status".to_string()),
+            ScType::Error => Some("Error".to_string()),
```

Comment:
These are example inputs, so perhaps an Err is better. Though these are only seen when used and an Error as input is likely never to happen.